tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+skipsdist = true
+envlist = tests
+
+# These are the tests that run in CI. They can be run locally in a redislabs/redisgears:edge docker
+[testenv:tests]
+commands_pre =
+    pip install --upgrade pip
+    {envdir}/bin/python -m pip install poetry
+    {envdir}/bin/python -m pip install RLTest==0.5
```

Comment:
```
"RLTest>=0.5.0"
```
Isn't this  to keep within the same patch set?

Reply:
Fixed